packages/react-core/src/components/Dropdown/examples/Dropdown.md

```diff
@@ -63,3 +63,11 @@ To provide users with more context about a `<DropdownItem>`, pass a short messag
 ```ts file="./DropdownWithDescriptions.tsx"
 
 ```
+
+### Split button
+
+Description of split button
```

Comment:
```suggestion
To combine a checkbox or other control with a dropdown menu, use a split button.

A `<MenuToggle>` can be rendered as a split button via `splitButtonItems`. Elements to be displayed before the dropdown toggle button (like the `<MenuToggleCheckbox>`) must be included in the `splitButtonItems`.

If the dropdown menu closes upon selection, you will need to manually shift focus back to the toggle element after a user selects an item from the menu. 

```
wdyt about this?

Reply:
🚢 it

@Mash707 if this looks good to you can you make this quick update?